build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
```

Comment:
Okay not to change in our repo.   I'll open a PR in CoreFx next time I need to hand-edit the value.  CoreFx needs to make this a configurable property.